paper.bib

```diff
@@ -236,9 +235,59 @@ @misc{ flask:tutorial
           note = "[Online; accessed 25-June-2017]"
       }
 
+@book{davenport1998working,
+  title={Working knowledge: How organizations manage what they know},
+  author={Davenport, Thomas H and Prusak, Laurence},
+  year={1998},
+  publisher={Harvard Business Press}
+}
 
 
+@misc{tiobe,
+    author="TIOBE software BV",
+    title="TIOBE Index for February 2018",
+    howpublished="\url{https://www.tiobe.com/tiobe-index/}",
+    note="[Online; accessed 28-February-2018]"
+}
 
+@ARTICLE{Zeng,
+author={Liangzhao Zeng and B. Benatallah and A. H. H. Ngu and M. Dumas and J. Kalagnanam and H. Chang},
+journal={IEEE Transactions on Software Engineering},
+title={QoS-aware middleware for Web services composition},
+year={2004},
+volume={30},
+number={5},
+pages={311-327},
+keywords={Internet;electronic commerce;integer programming;middleware;quality of service;B2B;QoS;Web manual interaction;Web services;integer programming;middleware platform;online business-to-business collaboration;quality of service;service composition;Availability;Computer Society;Computer science;Financial management;Linear programming;Middleware;Online Communities/Technical Collaboration;Quality of service;Web and internet services;Web services;65;Web services;integer programming.;quality of service;service composition},
+doi={10.1109/TSE.2004.11},
+ISSN={0098-5589},
+month={May},
+}
 
+@inproceedings{Kaminski:2006:DAW:1137677.1137694,
```

Comment:
are these referred from the text? 
then we should use the key naming convention
as proposed here: https://github.com/mircealungu/mir-biblio/blob/master/aslan.bib.md